src/ol/interaction/Pointer.js

```diff
@@ -193,7 +193,10 @@ class PointerInteraction extends Interaction {
       const event = mapBrowserEvent.originalEvent;
 
       const id = event.pointerId.toString();
-      if (mapBrowserEvent.type == MapBrowserEventType.POINTERUP) {
+      if (
+        mapBrowserEvent.type == MapBrowserEventType.POINTERUP ||
```

Comment:
This change won‘t make a difference, because `isPointerDraggingEvent()` prevents this code path from being entered on `pointercancel`.